dockerfiles/splash/provision.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+usage() {
+    cat <<EOF
+Splash docker image provisioner.
+
+Usage: $0 COMMAND [ COMMAND ... ]
+
+Available commands:
+usage -- print this message
+prepare_install -- prepare image for installation
+install_deps -- install system-level dependencies
+install_builddeps -- install system-level build-dependencies
+install_python_deps -- install python-level dependencies
+remove_builddeps -- remove build-dependencies
+remove_extra -- remove files that are unnecessary to run Splash
+
+EOF
+}
+
+prepare_install () {
+    # Prepare docker image for installation of packages, docker images are
+    # usually stripped and aptitude doesn't work immediately.
+    sed 's/main$/main universe/' -i /etc/apt/sources.list && \
```

Comment:
The short answer is that it is not guaranteed to work inside functions. The long answer I have found is here: http://lists.gnu.org/archive/html/bug-bash/2012-12/msg00093.html

I suppose I could drop the functions, however...